core/src/main/scala/org/http4s/blaze/channel/ChannelHead.scala

```diff
@@ -47,6 +47,6 @@ object ChannelHead {
     "Connection reset", // Found on Linux, Java 13
     "An existing connection was forcibly closed by the remote host", // Found on Windows
     "Broken pipe", // also found on Linux
-    "The specified network name is no longer available.\r\n" // Found on Windows NIO2
+    "The specified network name is no longer available.\r\n", // Found on Windows NIO2
```

Comment:
Trailing comma of Schrodinger. Never know will the next minor release of Scalafmt brings it or not.

Reply:
I'm glad to see them back!